### DISCUSSION TITLE
Machine Learning for automatic categorization

### DISCUSSION THREAD

**Discussion by fariasfc** [firefly-iii/firefly-iii#7598] [General]
Hello! First of all, thanks for this amazing tool! Started using it and I am really liking!

I would like to know if we can perform automatic categorization whenever a new transaction is registered using Natural Language Processing.

There are simple ML models that can do a decent job on that.

**Comment by JC5** [comment]
Check out: https://github.com/bahuma20/firefly-iii-ai-categorize
